### PR TITLE
en: update location label description for pd

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update location labels description for pd.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [x] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/161
- Other reference link(s):

### CODE REVIEW

File: en/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -197,11 +197,12 @@ To add the data disaster recovery feature in Kubernetes:
 
 1. Set the label collection of topological location for PD
 
+    Replace the `location-labels` information in the `pd.config` with the label collection that describes the topological location on the nodes in the Kubernetes cluster.
```
Comment (DanielZhangQD) [inline en/configure-a-tidb-cluster.md:200]:
```suggestion
    Configure the `location-labels` in the `pd.config` with the labels that describe the topology on the nodes in the Kubernetes cluster.
```

File: en/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -197,11 +197,12 @@ To add the data disaster recovery feature in Kubernetes:
...
+
     > **Note:**
     >
-    > Except for `kubernetes.io/hostname`, PD currently does not support labels with `/` in the name.
-
-    Replace the `location-labels` information in the `pd.config` with the label collection that describes the topological location on the nodes in the Kubernetes cluster.
+    > * For PD versions < v3.0.9, the `/` in the label name is not supported.
+    > * If you configure `hostname` in the `location-labels`, TiDB Operator will get the value from the `kubernetes.io/hostname` in the node label.
```
Comment (DanielZhangQD) [inline en/configure-a-tidb-cluster.md:205]:
```suggestion
    > * If you configure `hostname` in the `location-labels`, TiDB Operator will get the value from the `kubernetes.io/hostname` label on the node.
```

File: en/configure-a-tidb-cluster.md [hunk 3]
```diff
@@ -197,11 +197,12 @@ To add the data disaster recovery feature in Kubernetes:
 
 1. Set the label collection of topological location for PD
 
+    Configure the `location-labels` in the `pd.config` with the labels that describe the topology on the nodes in the Kubernetes cluster.
```
Comment (lilin90) [inline en/configure-a-tidb-cluster.md:200]: `with` has multiple meanings. Let's use a more accurate word.
```suggestion
    Configure `location-labels` in the `pd.config` file using the labels that describe the topology on the nodes in the Kubernetes cluster.
```

**Comment by ran-huang** [comment]
@lilin90 PTAL. Thanks~

**Pull request merged**